webpack/loaders.js

```diff
@@ -9,6 +9,11 @@ module.exports = [
         loaders: ['react-hot', 'babel']
```

Comment:
When using with transpiling loaders (like babel-loader), make sure they are in correct order (bottom to top). Otherwise files will be check after being processed by babel-loader
https://github.com/MoOx/eslint-loader